src/lib/inactivity.js

```diff
@@ -0,0 +1,16 @@
+export const inactivityTime = () => {
+  let time;
+  window.onload = resetTimer;
+  document.onmousemove = resetTimer;
```

Comment:
Suggestion: 
- prefer addEventListener to not override existing eventListeners
- maybe we need to handle pointer events because there is no mouse on the museum tablet